app/views/catarse_bootstrap/projects/_dashboard_user_settings.html.slim

```diff
@@ -65,40 +67,38 @@
                       = form.label :phone_number, class: 'field-label fontweight-semibold force-text-dark'
                       = form.input_field :phone_number, as: :string, class: 'positive'
           .w-form.card.card-terciary
-            = form.simple_fields_for :bank_account do |bform|
-              .fontsize-base.fontweight-semibold= t('.account_title')
-              .fontsize-small.u-marginbottom-20= t('.account_subtitle')
-              .divider.u-marginbottom-20
-              .w-row
-                .w-col.w-col-6.w-sub-col
-                  = bform.label :bank_id, class: 'field-label fontweight-semibold force-text-dark'
-                  = bform.input_field :bank_id, as: :select, collection: Bank.to_collection, class: 'w-select positive'
-                .w-col.w-col-6
-                  .w-row
-                    .w-col.w-col-6.w-col-small-6.w-col-tiny-6.w-sub-col-middle
-                      = bform.label :agency, class: 'field-label fontweight-semibold force-text-dark'
-                      = bform.input_field :agency, as: :string, class: 'positive'
-                    .w-col.w-col-6.w-col-small-6.w-col-tiny-6
-                      = bform.label :agency_digit, class: 'field-label fontweight-semibold force-text-dark'
-                      = bform.input_field :agency_digit, as: :string, class: 'positive'
-              .w-row
-                .w-col.w-col-6.w-sub-col
-                  .w-row
-                    .w-col.w-col-6.w-col-small-6.w-col-tiny-6.w-sub-col-middle
-                      = bform.label :account, class: 'field-label fontweight-semibold force-text-dark'
-                      = bform.input_field :account, as: :string, class: 'positive'
-                    .w-col.w-col-6.w-col-small-6.w-col-tiny-6
-                      = bform.label :account_digit, class: 'field-label fontweight-semibold force-text-dark'
-                      = bform.input_field :account_digit, as: :string, class: 'positive'
-                .w-col.w-col-6
-                  = bform.label :owner_name, class: 'field-label fontweight-semibold force-text-dark'
-                  = bform.input_field :owner_name, as: :string, class: 'positive'
-              .w-row
-                .w-col.w-col-6.w-sub-col
-                  = bform.label :owner_document, class: 'field-label fontweight-semibold force-text-dark'
-                  = bform.input_field :owner_document, as: :tel, class: 'positive'
-                .w-col.w-col-6
-    
+            .fontsize-base.fontweight-semibold= t('.account_title')
+            .fontsize-small.u-marginbottom-20= t('.account_subtitle')
+            .divider.u-marginbottom-20
+            .w-row
+              .w-col.w-col-6.w-sub-col
+                = form.label :bank_id, class: 'field-label fontweight-semibold force-text-dark'
```

Comment:
@adrianob try generate the form fields using form.input with wrappers because if we use input_field e can't track error style on field.